src/screens/onboarding/Onboarding.js

```diff
@@ -55,7 +55,7 @@ class Onboarding extends React.Component {
       // Otherwise, check for invite in query string
       const inviteToken = qs.parse(location.search, {
         ignoreQueryPrefix: true
-      }).token;
+      }).invite;
```

Comment:
```suggestion
      }).token;
```


shouldn't this be .token? It works when I do .token, but not .invite

Reply:
i switched it to `invite`